scaleway/resource_k8s_pool_test.go

```diff
@@ -460,7 +462,7 @@ func testAccCheckScalewayK8SPoolServersAreInPrivateNetwork(tt *TestTools, cluste
 			}
 
 			server, err := instanceAPI.GetServer(&instance.GetServerRequest{
-				Zone:     zone,
+				Zone:     scw.Zone(providerIDSplit[3]),
```

Comment:
Could you add a comment to give an example of the providerID format ?

Reply:
Done -> line 460